templates/1_set_up_toffy.ipynb

```diff
@@ -15,10 +15,10 @@
    "id": "e36293c5-aa89-4029-a3fa-e8ea841bb8b5",
```

Comment:
<p>Harmonize with readme description, link to it here instead</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>